### DISCUSSION TITLE
reCapcha request always return {"message":"Invalid reCAPTCHA response"} in refresh metadata api

### DISCUSSION THREAD

**Discussion by ThaiKhim** [blockscout/blockscout#11237] [Q&A]
I need help setting up reCapcha for docker compose. I tried creating and filling capcha in `NEXT_PUBLIC_RE_CAPTCHA_V3_APP_SITE_KEY` but it always returns:`"message":"Invalid reCAPTCHA response"`

**Comment by GimluCom** [comment]
Have you also set the `RE_CAPTCHA_V3_SECRET_KEY` and `RE_CAPTCHA_V3_CLIENT_KEY` ENVs? [docs.blockscout.com](https://docs.blockscout.com/setup/env-variables/backend-env-variables#csv-export)
